dev/consdeploy.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python
+# -*- coding:utf-8 -*-
+
+import os, json, urllib2, subprocess, time, datetime, sys
+
+class GitHubAccess(object):
+    def __init__(self):
+        # define github access token mix with - or space to ignore from github security
+        self.token = "20-822 144 dabd-43bec-ab7cf5 334b530 6-e05ed-d cf 0"
+        self.repo = "test-blog-compiled"
+        self.api = "https://api.github.com/repos"
+        self.gitUser = "devfans"
+        return None
+    # get tags api url
+    def getTagsUrl(self):
```

Comment:
I deviate the way because I think mixCase convention seems more readable and keeps a short string:)